crates/runtime/src/instance/allocator.rs

```diff
@@ -59,6 +59,9 @@ pub struct InstanceAllocationRequest<'a> {
 
     /// The pointer to the stack map registry to use for the instance.
     pub stack_map_registry: *mut StackMapRegistry,
+
+    /// The resource limiter to use for the instance.
+    pub limiter: Option<Arc<dyn ResourceLimiter>>,
```

Comment:
If this ends up being stored in individual tables/memories then this could probably turn into `&'a Arc<dyn ...>`

Reply:
I'll change this, no reason to clone it for the request until a memory / table needs it.